plugins/wysiwygarea/plugin.js

```diff
@@ -716,3 +800,33 @@ CKEDITOR.config.disableNativeSpellChecker = true;
  * @param {CKEDITOR.editor} editor This editor instance.
  * @param {CKEDITOR.dom.element} data The element being added.
  */
+
+/**
+ * Document observation starting point. Native DOM object.
+ *
+ * To recreate editor `wysiwygarea` iframe after reattach editor to DOM:
+ *
+ * * make sure detachable element is placed any nested depth under observable element.
+ * * make sure editor is placed any nested depth under detachable element.
+ *
+ * See overview of HTML hierarchy below:
```

Comment:
I think it's important to mention that it's just a pseudocode and not actual HTML tags 🙃